package.json

```
@@ -21,7 +21,7 @@
     "doctor": "./scripts/doctor.js",
     "generate-cities-cache": "node scripts/generate-cities-cache.js",
     "generate-cities-objc": "node scripts/generate-cities-objc.js",
-    "ios": "react-native run-ios",
+    "ios": "cd ios; xcodebuild build -workspace Artsy.xcworkspace -scheme Artsy; cd ..",
```

```suggestion
    "ios": "react-native run-ios --udid="$(xcrun simctl list | awk -F'[()]' '/(Booted)/ { print $2 }')",
```

just adding here the suggestion I offered, so its ready. please check if that is easier and accept or close 👍

---

ew those `\"` lol. i dont like package.json scripts a lot 😅. if we like this, at some point I would just move it to a `./scripts/start-ios` or something.

---

@pvinis will this not necessitate that you have a simulator already running?

---

Sorry, Just saw your comment. Noted

---

yep. we can add `yarn open-sim` in front of it.